airbyte-integrations/connectors/source-sentry/source_sentry/manifest.yaml

```diff
@@ -1,105 +1,149 @@
-version: "0.29.0"
+version: 0.57.0
+type: DeclarativeSource
+
 definitions:
-  page_size: 50
   schema_loader:
     type: JsonFileSchemaLoader
-    file_path: "./source_sentry/schemas/{{ parameters.name }}.json"
-  selector:
+    file_path: "./source_sentry/schemas/{{ parameters['name'] }}.json"
+  authenticator:
+    type: BearerAuthenticator
+    api_token: "{{ config['auth_token'] }}"
+  requester:
+    type: HttpRequester
+    url_base: "https://{{ config['hostname'] }}/api/0/"
+    http_method: GET
+    request_headers: {}
+    authenticator:
+      $ref: "#/definitions/authenticator"
+  record_selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: []
+  filter_record_selector:
     type: RecordSelector
     extractor:
       type: DpathExtractor
       field_path: []
-  requester:
-    type: HttpRequester
-    url_base: "https://{{ config.hostname }}/api/0/"
-    http_method: "GET"
-    authenticator:
-      type: "BearerAuthenticator"
-      api_token: "{{ config.auth_token }}"
   paginator:
     type: DefaultPaginator
-    page_size: "#/definitions/page_size"
-    limit_option:
-      inject_into: "request_parameter"
-      field_name: ""
     page_token_option:
       type: RequestOption
-      inject_into: "request_parameter"
-      field_name: "cursor"
+      inject_into: request_parameter
```

Comment:
Out of curiosity: what formatting tool are you using? I want to make sure we will setup our tooling so formatting is consistent across the team

Reply:
I used manual + IDE + airbyte-ci formatting.  `page_token_option: ` was added and formatted by connector builder.